### PR TITLE
Detail view / Delete function

### PR DESCRIPTION
**Pull request by DerElchausdemAll**
## Motivation

- added delete function, but it is only working after refreshing the page
- fixed a mistake in my GET method
- added a detailed view of the items (click on the card title)
- added some color to my navbar

### CODE REVIEW

File: src/pages/details/[id].js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+	return (
+		<Layout>
+			<Head>
+				<title key="title">About</title>
+				<meta key="description" name="description" content="About" />
+			</Head>
+
+			<h1>{item.title}</h1>
+			<img src={item.image} alt={item.title} />
```
Comment (br3zn) [inline src/pages/details/[id].js:44]: Use [next/image](https://nextjs.org/docs/basic-features/image-optimization) instead to benefitt from Next.js superpowers like automatic image optimisation.

**Review by br3zn — CHANGES_REQUESTED** [review]
Change the img tag to a next/image component. Rest looks fine 👍🏼

File: src/pages/details/index.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+// 							<ItemsCard item={item} />
+// 						</Grid>
+// 					))}
+// 				</Grid>
+// 			</Container>
+// 		</Layout>
+// 	);
+// };
+
+// export default Test;
```
Comment (doemser) [inline src/pages/details/index.js:67]: you need this unused code?

**Review by doemser — APPROVED** [review]
looks good :) <= was what I wrote, before I saw that all of your tests have failed :o a little bit to hasty approved :/

**Pull request merged**